.vscode/extensions.json

```diff
@@ -0,0 +1,69 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		//General recommandations
+		"aaron-bond.better-comments", 	     		//Better Comments
+		"esbenp.prettier-vscode",		     		//Prettier
+		"streetsidesoftware.code-spell-checker",	//Code Spell Checker
+		"dbaeumer.vscode-eslint",		 		  	//ESLint
+		"wayou.vscode-todo-highlight",			    //Todo Highlight	
+
+		// To contribute with Golang
+		"golang.go",                         		// Go language support
+
+		//To contribute with python
+		"ms-python.python",	                 		//Python
+		"ms-python.black-formatter",         		//To format the code with black
+		"slightc.pip-manager",               		//To install pip-free packages
+		"ms-python.isort",							//To sort the imports
+		"the-compiler.python-tox",					//To run the tests
+		"ms-python.pylint",							//To run the linter
+
+		//To contribute with docker
+		"ms-azuretools.vscode-docker",       		//Docker support
+		"ms-vscode-remote.remote-containers",		//To run the containers remotely with docker-machine and docker-compose 
```

Comment:
We don't need those, we don't have any docker  running during development

Reply:
ok, I will remove it